circle.yml

```diff
@@ -8,9 +8,10 @@ dependencies:
   pre:
     # update locally with:
     # openssl aes-256-cbc -e -in ./buildres/jabref-cert-2016.p12 -out jabref-cert-2016.p12.enc -k {PASSWORD}
-    - openssl aes-256-cbc -d -in ./buildres/jabref-cert-2016.p12.enc -out ./buildres/jabref-cert-2016.p12 -k $CERTIFICATE
+    #- openssl aes-256-cbc -d -in ./buildres/jabref-cert-2016.p12.enc -out ./buildres/jabref-cert-2016.p12 -k $CERTIFICATE
```

Comment:
I would also delete `./buildres/jabref-cert-2016.p12.en` as we do not need this file any more.